pkg/proxy/benchmarking/benchmark.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -o errexit
+set -o nounset
+set -o pipefail
```

Comment:
nit: you can collapse these to a single line single `set` call. `-e` and `-o errexit` are also redundant

Reply:
new to shell scripting, learning new things. will update this